test/TradeTrustERC721.js

```diff
@@ -65,4 +69,84 @@ contract("TradeTrustErc721", accounts => {
       /VM Exception while processing transaction: revert ERC721: transfer caller is not owner nor approved/
     );
   });
+
+  describe("Surrendered TradeTrustERC721 Work Flow", () => {
+    let tokenRegistryInstanceWithShippingLineWallet;
+    let tokenRegistryAddress;
+    let escrowInstance;
+    let escrowInstanceAddress;
+    let escrowInstance1;
+    let escrowInstanceAddress1;
+
+    beforeEach("", async () => {
+      // Setup in accordance to workflow, starting test after the point of surrendering ERC721 Token
+      tokenRegistryInstanceWithShippingLineWallet = await Erc721.new("foo", "bar", { from: shippingLine });
+      tokenRegistryAddress = tokenRegistryInstanceWithShippingLineWallet.address;
+      escrowInstance = await TitleEscrow.new(tokenRegistryAddress, beneficiary1, beneficiary1, ZERO_ADDRESS, {
+        from: beneficiary1,
+      });
+      escrowInstanceAddress = escrowInstance.address;
+      escrowInstance1 = await TitleEscrow.new(tokenRegistryAddress, beneficiary1, beneficiary1, ZERO_ADDRESS, {
+        from: beneficiary1,
+      });
+      escrowInstanceAddress1 = escrowInstance1.address;
```

Comment:
This contract doesn't need to know the existence of escrow contract right?

Reply:
Can simply use another account that is not the minter

Reply:
Yup, however in this case we wanted to test the entire flow of the surrender procress accurately which meant transfering to title escrow, this way we can catch errors like using non-safe functions within the new functions

Reply:
I disagree in this case because it's really more of an unit test here. Having the title escrow kind of slow down the test and make the unit test less readable also.